### ISSUE TITLE
Change Request: Push Notification Migrations from VEText to VA Notify

### ISSUE THREAD

**Issue by kbelikova-oddball** [department-of-veterans-affairs/va-mobile-app#10147]
### 1.) Requestor's Name

Ksenia Belikova / Kyle MacMillan

### 2.) Requestor's Email

ksenia.belikova@va.gov kyle.macmillan@va.gov

### 3.) VA Team

VA Notify

### 4.) What is the name of the project or change?

Mobile Registration through Va Notify

### 5.) Please explain what problem you are trying to solve.

VEText currently handles all mobile registrations, we'd like to migrate registration of devices to VA Notify. 
As a first step, we'd like to also receive a registration request in addition to VEText. And we'd like to know what the appropriate response to mobile team's https request should be.

### 6.) Please explain how this change would work.

1. VA Mobile team sends Va Notify requests in parallel with VEText, so there are no disruptions.
2. VA Notify will respond with a mock value of their choosing.
3. This  would probably need to be feature-flagged until we all are ready to migrate.
4. Future work will involve a proper response and transitioning to using VA Notify's response (and dropping VEText).

### 7.) Features Impacted

Sending push notifications

### 8.) Please include all relevant documentation links for this change request.

[Push Notifications Brief Document](https://dvagov.sharepoint.com/:w:/r/sites/VANotify-internal/_layouts/15/Doc.aspx?sourcedoc=%7BE400C098-9303-40F4-8A7B-617497F597C7%7D&file=Push%20Notifications%20Brief.docx&action=default&mobileredirect=true)

[Proposed Diagrams ](https://github.com/department-of-veterans-affairs/vanotify-team/tree/main/Architecture/push-notifications/images)

### 9.) Does your team have all the necessary materials (documents, designs, processes, analytics, APIs, etc) in place for Mobile to make these changes right now?

No

### 10.) Has this change been discussed with VAHB POs (Ryan or Don)?

Don - yes

### 11.) Has this change been agreed upon by all impacted VA Stakeholders?

Yes

### 12.) What is your ideal due date for this change request?

End of Q4 2024

**Comment by kbelikova-oddball** [comment]
Hello! I was wondering if there are any updates on this.

**Comment by BeccaTupaj** [comment]
This work is being tracked on the Core Team project board per #10467  . Closing this Change Request. cc: @kbelikova-oddball @davidconlon
